### DISCUSSION TITLE
Non active branches

### DISCUSSION THREAD

**Discussion by selfuryon** [nix-community/ethereum.nix#168] [Show and tell]
Hello! 

What do you think about enabling auto-delete branches after merge? 
See a lot of merged branches here
![image](https://user-images.githubusercontent.com/2993917/225163078-8bee7306-a14b-4b01-9601-05475a3a56a5.png)

**Comment by aldoborrero** [comment]
I haven't seen this until now! Yep, it's still not enabled we should do it 💯  @selfuryon

**Reply by selfuryon** [reply]
@brianmcgee What do you think about that? Can you enable that? 

I also suggest enabling squashing with commit details, it's collect all commit details in the commit description, easy to understand what's going on.
![image](https://github.com/nix-community/ethereum.nix/assets/2993917/7a799c6f-e3e7-4fb0-a3fe-cf6c0588d136)

**Reply by aldoborrero** [reply]
We have now enabled both options.
